main.go

```diff
@@ -0,0 +1,176 @@
+//go:generate go run pkg/codegen/cleanup/main.go
+//go:generate /bin/rm -rf pkg/generated
```

Comment:
the `rm ....` line can be removed, already has this logic in `pkg/codegen/cleanup/main.go`